CODEOWNERS

```diff
@@ -6,8 +6,8 @@
 /core @smartcontractkit/foundations
 
 # Chains
-/common @smartcontractkit/integrations
-/core/chains/ @smartcontractkit/integrations
+/common @smartcontractkit/bix-framework
+/core/chains/ @smartcontractkit/bix-framework
```

Comment:
Should we assign solana/starknet/cosmos ownership to BUILD?

Reply:
afaik there isn't any nonEVM under /core/chains anymore - that's all been shifted to their own repos with the LOOP work